### PR TITLE
Update our Issue templates and Pull request templates

### PR DESCRIPTION
**Pull request by larson-carter**
- I updated the bug reports template
- I updated the feature request template
- I created the Pull Request template

This will allow for us to set higher priorities amongst these issues. It also will allow us to debug issues better than before.

I also make the pull request templates more structured.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,32 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a report to help us improve ohmyzsh!
+title: ''
+labels: ''
+assignees: ''
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The person making the PR will fill them in?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,38 +1,32 @@
...
 ---
 
-<!--
-Fill this out before posting. You can delete irrelevant sections, but
-an issue where no sections have been filled will be deleted without comment.
--->
-
 **Describe the bug**
-<!-- A clear and concise description of what the bug is. -->
+A clear and concise description of what the bug is including themes.
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, I'll correct it later.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,38 +1,32 @@
...
+title: ''
+labels: ''
+assignees: ''
 
 ---
 
-<!--
-Fill this out before posting. You can delete irrelevant sections, but
-an issue where no sections have been filled will be deleted without comment.
--->
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, I'll add it back.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,38 +1,32 @@
...
-
 **Describe the bug**
-<!-- A clear and concise description of what the bug is. -->
+A clear and concise description of what the bug is including themes.
 
 **To Reproduce**
-<!--
 Steps to reproduce the behavior:
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok, I'll add it when I go in and make some changes later.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -1,38 +1,32 @@
...
 **Expected behavior**
-<!-- A clear and concise description of what you expected to happen. -->
+A clear and concise description of what you expected to happen.
 
-**Screenshots or recordings**
-<!--
-If applicable, add screenshots or record an asciinema session (https://asciinema.org/)
-to help explain your problem.
--->
+**Screenshots and or Recordings**
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I will fix later.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -1,38 +1,32 @@
...
+If applicable, add screenshots to help explain your problem. Also you can record an asciinema session (https://asciinema.org/)
 
-**System:**
- - OS: [e.g. macOS]
- - Zsh version [e.g. 5.6]
+**Desktop (please complete the following information):**
+ - OS / Distro: [e.g. Arch Linux, macOS]
+ - Latest ohmyzsh Update?: [e.g. Yes/No]
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Great!

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -11,16 +13,17 @@ an issue where no sections have been filled will be deleted without comment.
 -->
 
 **Is your feature request related to a particular plugin or theme? If so, specify it.**
-<!-- The name of the plugin or theme you'd like us to improve. -->
+The name of the plugin or theme that you would like us to improve. [...]
 
 **Is your feature request related to a problem? Please describe.**
-<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ok, I'll look at another way to state it.

File: PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+_Delete where appropriate and specify the repositories._  
+
+Yes / No
+
+<br>
+
+<!-- Question 3 -->
+**3. Repositories (ignore if your answer to the previous question was 'No'):**
+- ohmyzsh-bot (https://github.com/ohmyzsh/ohmyzsh-bot)
+- github-actions (https://github.com/ohmyzsh/github-actions)
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: Ok, we can talk about it on discord. I was mainly gearing it to when that guy makes a PR about our github actions.

File: PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+
+<br>
+
+
+## Standards Checklist
+
+- [ ] My code is meaningful and **comments**.
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: I was meaning to say please give any input in the comments section. Also the code is actually meaningful. I will fix.

File: PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] My code follows ohmyzsh principles.
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: Yes, I was referring to code style. Before we merge this PR we also need to update the wiki, then I'll revisit this checklist. Basically schemes to follow.

File: PULL_REQUEST_TEMPLATE.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] My code is, to the best of my ability, efficient and does not waste processor time.
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: True, I will change.

File: PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+<!-- Question 3 -->
+**3. Repositories (ignore if your answer to the previous question was 'No'):**
+- ohmyzsh-bot (https://github.com/ohmyzsh/ohmyzsh-bot)
+- github-actions (https://github.com/ohmyzsh/github-actions)
+
+
+<br>
+
+
+## Standards Checklist
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: Great, I'll fix it later.

File: PULL_REQUEST_TEMPLATE.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
+## Pull Request: [Put a PR title here...]
```
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: Interesting, I'll look into that also.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcornella) [reply .github/ISSUE_TEMPLATE/bug_report.md]: These fields are used to automatically set the fields in the issue.

For instance, if opening a bug report with labels: 'Type: bug', the issue will be labeled 'Type: bug'. Or if assignees is set, the issue will be automatically assigned to the users in the assignees field. The title value is used to prepopulate the title field, but the user can change it. This is the only field that the user can touch.

By default GitHub assumes they're blank, we don't need to add them to the template.

[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (larson-carter) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed

File: PULL_REQUEST_TEMPLATE.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (larson-carter) [reply PULL_REQUEST_TEMPLATE.md]: Ok, we need to make a structured code style, we also need to update the wiki to reflect these changes.

**Comment by larson-carter** [comment]
@mcornella We are ready to merge. We just need to rebase to the master branch!

**Comment by mcornella** [comment]
Merged into master. Sorry for taking so long 😅

**Pull request closed without merging**